setup.R

```diff
@@ -1,10 +1,6 @@
-# Install civis R client
-options(unzip='internal');
-devtools::install_github('civisanalytics/civis-r', ref = 'v2.1.0', upgrade_dependencies = FALSE);
-
 # Install R Kernel for Jupyter
 install.packages(c('IRdisplay', 'pbdZMQ'))
-devtools::install_github('IRkernel/IRkernel', ref = '1.0.0', upgrade_dependencies = FALSE);
+devtools::install_github('IRkernel/IRkernel', ref = '1.0.2')
```

Comment:
`upgrade_dependencies` is no longer an argument. The same behavior is retained using default arguments.

Upgrading IRkernal from `1.0.0 -> 1.0.2` should improve code completion.

Reply:
Do you not need to install the client itself anymore?

Reply:
Nope, it's coming from `datascience-r`.

Reply:
Ah, of course, got it.